SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialStatics.h

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Kismet/BlueprintFunctionLibrary.h"
+#include "SpatialStatics.generated.h"
+
+DECLARE_LOG_CATEGORY_EXTERN(LogSpatialStatics, Log, All);
+
+UCLASS()
+class SPATIALGDK_API USpatialStatics : public UBlueprintFunctionLibrary
+{
+    GENERATED_BODY()
+
+public:
+    /**
+     * Returns whether SpatialOS networking has been enabled or not.
+     */
+    UFUNCTION(BlueprintPure, Category = "SpatialOS")
```

Comment:
This may be the wrong place to ask this, but why have SpatialOS as a category and Spatial in all the names? Would it be better to use SpatialOS in the names too?